wandb/sdk/wandb_run.py

```diff
@@ -207,6 +207,15 @@ def start(self) -> None:
         self._network_status_thread.start()
         self._internal_messages_thread.start()
 
+    @staticmethod
+    def _stop_check(
```

Comment:
The name `_stop_check` to me implies something more like "raise if not stopped" or return true if stopped. Something more like `_abandon`, perhaps? Stylistically I'm also not sure why this is a static method rather than a plain function, but that's not even a nit, just a question.

Reply:
static just to have it part of the same `RunStatusChecker` class. 
if you have suggestions for a better name, i can rename it, also can add some docstring if it helps, just though it is so  trivial  we can skip it

Reply:
renamed to `_abandon_status_check`